src/client/app/components/admin/PreferencesComponent.tsx

```diff
@@ -154,7 +207,11 @@ export default function PreferencesComponent() {
 					value={localAdminPref.defaultMeterMinimumValue}
 					onChange={e => makeLocalChanges('defaultMeterMinimumValue', e.target.value)}
 					maxLength={50}
```

Comment:
Maybe take a look at src/client/app/components/meters/CreateMeterModalComponent.tsx where the min & max can constrain the values input by the arrows. You still need the checks but it helps some and would be parallel for similar items. This applies to all the min/min checks on this page.